decibel/theme.json

```diff
@@ -31,9 +31,14 @@
 				}
 			]
 		},
+		"custom": {
+			"spacing": {
+				"outer": "clamp(1.25rem, 0.32692307692307687rem + 1.9230769230769231vw, 2.25rem)"
+			}
+		},
 		"layout": {
-			"contentSize": "620px",
-			"wideSize": "1000px"
+			"contentSize": "664px",
```

Comment:
I inferred this value from the main column on the home page designs, since there is no explicit single template designs from which to pull this value.